src/classes/RD2_DataMigrationMapper.cls

```diff
@@ -317,6 +321,13 @@ public class RD2_DataMigrationMapper {
         }
     }
 
+    private void setFixedToOpenDuringDryRun(npe03__Recurring_Donation__c rd) {
+        if (!new RD2_RecurringDonation(rd).isFixedLength()) {
+            return;
+        }
+        rd.RecurringType__c = RD2_Constants.RECURRING_TYPE_OPEN;
```

Comment:
If I'm not mistaken, even in DryRunMode, if the "Fixed" picklist option exists this should set RecurringType to "Fixed". We're only using "Open" in the event that the picklist value does not exist or is inactive.

Reply:
Yes.  Thanks for catching that!